File: samples/client/petstore/java/retrofit2rx2/src/main/java/org/openapitools/client/api/DefaultApi.java
package org.openapitools.client.api;

import org.openapitools.client.CollectionFormats.*;

import io.reactivex.Observable;
import io.reactivex.Completable;
import retrofit2.http.*;

import okhttp3.RequestBody;
import okhttp3.ResponseBody;
import okhttp3.MultipartBody;

import org.openapitools.client.model.ExampleResponse;

import java.util.ArrayList;
import java.util.HashMap;
import java.util.List;
import java.util.Map;
import java.util.Set;

public interface DefaultApi {
  /**
   * 
   * dummy
   * @return Observable&lt;ExampleResponse&gt;
   */
  @GET("example/someMethod")
  Observable<ExampleResponse> dummy();
    

}

File: samples/client/petstore/kotlin-retrofit2-kotlinx_serialization/src/main/kotlin/org/openapitools/client/apis/DefaultApi.kt
package org.openapitools.client.apis

import org.openapitools.client.infrastructure.CollectionFormats.*
import retrofit2.http.*
import retrofit2.Call
import okhttp3.RequestBody
import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

import org.openapitools.client.models.ExampleResponse
import org.openapitools.client.models.Node

interface DefaultApi {
    /**
     * 
     * dummy
     * Responses:
     *  - 200: dummy
     *  - 201: dummy
     *
     * @return [Call]<[ExampleResponse]>
     */
    @GET("example/someMethod")
    fun dummy(): Call<ExampleResponse>

}
